tests/test_utils/api_connexion_utils.py

```diff
@@ -110,6 +110,11 @@ def delete_user(app, username):
             break
 
 
+def delete_users(app):
```

Comment:
Is there a reason to have this function under `api_connexion_utils` when its only used in the CLI tests?

Reply:
This was added with a view that to be used later in other suites where similar issues happens when users are created and never deleted.